norfair/drawing.py

```diff
@@ -223,7 +223,7 @@ def draw_boxes(frame, detections, line_color=None, line_width=None, random_color
 def draw_tracked_boxes(
     frame,
     objects,
-    line_color=None,
+    line_color=None,  
```

Comment:
The name of the parameter doesn't convey that it can be a tuple. It's confusing.

Maybe name it `line_colors` or just `colors`?

Or maybe renaming both to `border_colors` and `border_width` makes more sense?